### DISCUSSION TITLE
Sort categories

### DISCUSSION THREAD

**Discussion by PatheticMustan** [community/community#2867] [Discussions] [answered]
Currently, categories are only sorted from A-Z.

![image](https://user-images.githubusercontent.com/35382971/101563737-d1584500-3997-11eb-8167-504c50e91093.png)

It would be cool to be able to organize the categories in a certain way, or more filters (like Z-A, newest posts, most posts)...

**Comment by nijikokun** [comment]
Agree'd the first thing I did was try to re-order after it put "Feedback" above "General"

**Comment by evi-liu** [comment, marked as answer]
Thank you for the feedback! We definitely see value in being able to customize the order of categories. We are tracking this request and prioritizing within our backlog.

**Reply by volkantash** [reply]
Bir yıl boyunca bir değişiklik oldu mu? (GT: Has there been a change over the course of a year?)

**Reply by twisted-nematic57** [reply]
Still nothing. I don't want to sound impatient but is such a basic feature still not implemented after 2 years??

I just prepended an asterisk or exclamation mark before the section I wanted on top.

**Reply by Eric2-Praxinos** [reply]
Still nothing. I don't want to sound impatient but is such a basic feature still not implemented after 5 years??
